### ISSUE TITLE
Can't install on C++Builder 10.2 Tokyo with Fatal Error.

### ISSUE THREAD

**Issue by tsirorret** [graphics32/graphics32#26]
1. Set [Tools->Environment Options->Delphi Options->Library->32-bit Windows->Lib Path
2. Open [RX2->GR32_R.dpk] with  [Delphi Compiler->Output -C/C++ ->Generate all C++Builder files (including package libs)]
==>Build
{$IFDEF IMPLICITBUILDING This IFDEF should not be used by users}
[dcc32 Fatal Error] GR32_R.dpk(3): E2280 Unterminated conditional directive

**Comment by tsirorret** [comment]
you can find wrong word, I don't know why those are changed during compile.
[rrequires] --> [requires]
[ocontains]-->[contains]
[d.]-->[end.]

**Comment by CWBudde** [comment]
For many years we still miss a maintainer of this project for C++ Builder. This means it is not only untested, but also not really meant for C++ Builder out of the box.

I've just tested it in Delphi RX10.2 (Tokyo) and it works there. So it seems to be really an issue with C++ Builder.

If you find a way to make it work, feel free to post it here so that we can merge the changes here as well.

**Comment by micha137** [comment]
Please test with RX2 C++ Builder!

**Comment by tsirorret** [comment]
I just test it. It looks ok, but there are small erros.
For exam, I can't [SaveToFile], and can find colors like clRed32.
and so on.

**Comment by rgomezc** [comment]
I have tried with 10.1 Berlin and the only problem I found so far was when building the packages: for some reason (I believe after opening the project options and saving some changes there) the IDE duplicates some compiler options in the package source.

It might be that some of the _R packages source doesn't include the `{$R *.res}` define?

I kept getting the duplicated lines until I left the `{$R *.res}` define as the first line after `package GR32_R;` That line is not included in the package source as of the 1563c9a commit.

The error commented at the start of this thread, about IMPLICITBUILD only happened when the IDE was duplicating the lines. Removing the duplicates fixed that issue. 

I have been able to build and use Graphics32 in C++Builder 10.1 without problems so far, except for 2 Extended consts in GR32.hpp that make the compiler unable to produce a precompiled header using that hpp. Commenting those constants solves that issue.
